src/Equinor.ProCoSys.IPO.WebApi/Middleware/GlobalExceptionHandler.cs

```diff
@@ -74,9 +74,16 @@ public async Task InvokeAsync(HttpContext context)
                 _logger.LogDebug(message);
                 await context.Response.WriteAsync(message);
             }
+            catch (IpoSendMailException mex)
+            {
+                _logger.LogError(mex, "An exception occurred when sending email");
+                context.Response.StatusCode = (int)HttpStatusCode.InternalServerError;
+                context.Response.ContentType = "application/text";
+                await context.Response.WriteAsync("Something went wrong when sending email!");
+            }
             catch (Exception ex)
             {
-                _logger.LogError(ex, "An exception occured");
+                _logger.LogError(ex, "An exception occurred");
```

Comment:
Do we get the $"User with oid {_currentUserProvider.GetCurrentUserOid()} could not create outlook meeting for invitation {invitation.Id}.", ex
information as innerexception in Application Insights?